aries-core/src/js/components/core/Tiles/stories/simple.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Tile, Tiles } from 'aries-core';
+
+import { Grommet, Box, Text, Anchor } from 'grommet';
+import { hpeFont } from './prototype';
+
+export default {
+  title: 'Tiles',
+};
+
+const content = [
+  'Shimi dev - Shimi Shimi Coco Puff.  ',
+  'Taytay dev - All tiles in the Grid container are symetric.',
+  'Gliss dev - Responsive works better with Grid.',
+];
+
+const TileCollection = () =>
+  [0, 1, 2].map(index => (
```

Comment:
```suggestion
  content.map((item, index) => (
```

Reply:
Thinking you can simplify the mapping here so if more content were added, you wouldn't need to go back and adjust the array of [0, 1, 2]

Reply:
I would have agreed if this content is used more then once. Not sure it is critical in this case.

Reply:
Done.